sklearn/ensemble/_weight_boosting.py

```diff
@@ -969,6 +969,23 @@ class AdaBoostRegressor(RegressorMixin, BaseWeightBoosting):
 
         .. versionadded:: 0.24
 
+    See Also
+    --------
+    AdaBoostClassifier : A meta-estimator that begins by fitting a classifier
+        on the original dataset and then fits additional copies of the
+        classifier on the same dataset where the weights of incorrectly
+        classified instances are adjusted such that subsequent classifiers
+        focus more on difficult cases.
```

Comment:
I think on a page that already has a description of AdaBoost this is a bit too verbose. Maybek 

```suggestion
    AdaBoostClassifier : An AdaBoost classifier.
```
would be sufficient. Indicating that it's an estimator that is similar to this regressor but for classification.